src/System Application/App/User Settings/src/ApplicationUserSettings.Table.al

```diff
@@ -33,6 +33,15 @@ table 9222 "Application User Settings"
         {
             DataClassification = CustomerContent;
         }
+
+        /// <summary>
+        /// Specifies whether the legacy action bar is enabled.
+        /// </summary>
+        field(3; "Legacy Action Bar"; Boolean)
```

Comment:
"Use Legacy Action Bar"? otherwise it more sounds like an enum where I can choose which Legacy Action Bar I want.

Reply:
Could also be "Enable Legacy Action Bar", aligns better with the name of your page field.

Reply:
Keeping consistent with `Teaching Tips`. That doesn't specify what teaching tips either.